django/thunderstore/community/tasks.py

```diff
@@ -0,0 +1,18 @@
+import logging
+from typing import Optional
+
+from celery import shared_task
+
+from thunderstore.community.models import Community, CommunityAggregatedFields
+from thunderstore.core.settings import CeleryQueues
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task(queue=CeleryQueues.BackgroundTask)
+def update_community_aggregated_fields(batch_size: Optional[int] = None) -> None:
```

Comment:
Commenting here since it seemed the most appropriate: This task should probably be paired with a schedule. Schedules are db objects so you'll need to use a migration to insert it. Example here: https://github.com/thunderstore-io/Thunderstore/blob/master/django/thunderstore/repository/migrations/0023_add_cache_refresh_celery_schedule.py

Reply:
Added a schedule, as well as a data migration that creates the initial values.